src/ui/StepManager.js

```diff
@@ -56,7 +56,7 @@ StepManager.prototype.render = function () {
 StepManager.prototype.reset = function () {
   this.slider.setValue(0)
   this.currentStepIndex = 0
-  this.buttonNavigator.stepChanged(0)
+  this.buttonNavigator.reset()
```

Comment:
this clearly reset this view instead of moving to 0 and log 'trace not found' each time